README.md

```diff
@@ -4,28 +4,95 @@
+
+
+## Introduction
+
+We were approached by a professor at the University of Minnesota Morris to create an alternative announcement system to replace the use of sticky notes posted on a door. DoorBoard allows owners to create notes to display to their viewers, accessible from anywhere. This application is currently intended to be used to announce things like “running late due to traffic” or  “Office Hours are cancelled today, my child got sick.”
+
+Owners can create notes, edit notes,  pin written notes to the top of the page, trash notes, and view their trashcan of deleted notes to decide whether to permanently delete the notes or reuse them. Additionally, each note is timestamped with the data and time when the note was created or last edited. Owners also have access to a button which generates a PDF containing the URL of the viewer page for their DoorBoard, as well as a QR code linking to that same URL.
+Viewers can see the notes left by the owner and see the owner’s Google Calendar using the link on the viewer page.
+
+This project is made possible by the students and faculty of CSci 3601, Software Development and Design, at the University of Minnesota Morris, and a special thanks to Rachel Johnson, Professor at the University of Minnesota Morris for being our customer for this project. 
+
+Software Design Spring 2020 Iteration 3 <br>
+Team Name: 2016 Summer Olympic Athletes from the Cayman Islands <br>
+Team Members: Aaron Otten, Audrey Le Meur, Erik Rauer, Kyle Fluto, Luke Burdette, & Trent Merkins
+
+This repository began as team Nicolai Thärichen’s code for iteration 1 of the DoorBoard app. It was then expanded upon in iteration 2 by Team Rocket. It was then used as the code base for iteration 3 by the 2016 Olympic Athletes from the Cayman Islands.
+
+| Owner DoorBoard | Viewer Doorboard |
+|-----------------|------------------|
+| ![](owner_DB.jpg)    | ![](viewer_PG.jpg)    |
+
+## Libraries Used
+
+### Client Side
+* Angular 9 is used to build the client side.
+* TypeScript is the language most used on the client side.
+* Jasmine and Karma are used for testing.
+* Protractor is used for end to end testing
+* Gradle is used to tell Javalin to orchestrate the Client Side.
+* JsPDF (jspdf) is used for PDF generation.
+* Kjua (kjua-svk) is used for QR code generation.
+* Auth0 SDK for Single Page Applications (auth0-spa-js) is used for user authentication on the client side.
+* Node.js is used for asynchronous API.
+
+### Server Side
+* Java is the language used on the Server Side.
+* Javalin is used as a lightweight web framework.
+* JUnit is used for testing.
+* Mockito is used to mock various classes to assist in testing.
+* Gradle is used to build the project.
+
+### Database
+* MongoDB is used as our database for storing notes and owners.
+
+### Deployment
+* Docker is used to contain software in its own packages and still allow it to communicate with each other.
+* DigitalOcean is used as a hosting service.
+
```

Comment:
We should add a section about how Auth0 is used to securely handle the authentication of users.

Reply:
Again? I think it's mentioned in client side

Reply:
I was thinking that was just a reference to the Auth0 library that we use, not necessarily the fact that we use Auth0

Reply:
Could you add that?

Reply:
Yeah I can. Could you move the images to their own directory?

Reply:
Done